mobify/platform-scaffold
# Fix cart product parser

There was an error (see screenshot below) because of a missing value from the Cart store's list of products. This PR adds the missing value `available`.

![screen_shot_2017-05-29_at_11 32 42_am](https://cloud.githubusercontent.com/assets/734535/26560312/4bffb8b4-446a-11e7-9a52-06806ead05a6.png)

## Changes
- Add the required `availability` type value to the Cart's product parser

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add any product to your cart
- Reload the previewed app
- **Verify** that the afore mentioned error no longer appears in your console.
